src/v2/view-builder/views/PollView.js

```diff
@@ -0,0 +1,55 @@
+import { loc, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import polling from './shared/polling';
+import { MS_PER_SEC } from '../utils/Constants';
+
+const PollMessageView = View.extend({
+  template: hbs`
+    <div class="ion-messages-container">
+      {{i18n code="poll.form.message" 
+        bundle="login" arguments="countDownCounterValue" $1="<span class='strong'>$1</span>"}}
+    </div>
+    <div class="okta-waiting-spinner"></div>`
+  ,
+  getTemplateData () {
+    const countDownCounterValue = this.options;
+    return {
+      countDownCounterValue,
+    };
+  },
+});
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    title () {
+      return  loc('poll.form.title', 'login');
+    },
+
+    noButtonBar: true,
+    
+    initialize () {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.startPolling();
+    },
+
+    render () {
+      BaseForm.prototype.render.apply(this, arguments);
+      this.countDownCounterValue = Math.ceil(this.options.appState.getCurrentViewState().refresh / MS_PER_SEC);
```

Comment:
looks like `this.countDownCounterValue` has been set at `polling` prototype object.

Reply:
this one initializes `PollMessageView` value right before render completes.

Reply:
I meant `this.startPolling();` has create `this.countDownCounterValue` already.